languages/rust/oso/src/host/class.rs

```diff
@@ -56,34 +73,38 @@ impl Default for Class {
     }
 }
 
-impl<T> Class<T> {
+// TODO seems like the name is based on fully qualified name, so we may want to
+// require this to be specified.
```

Comment:
Yeah that's a good idea, it's defaulted for the proc macro so should be fine to error if it's not specified.

Reply:
Have a note to follow up on this as a separate PR, it will be easier if we refactor Class to have a builder.